test/modules/TestResultCoordinator/Reports/EdgeHubRestartTest/EdgeHubRestartMessageReport.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Reports.EdgeHubRestartTest
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    /// <summary>
+    /// This is a counting report to show test result counts, e.g. expect and match counts; and contains a list of unmatched test results.
+    /// </summary>
+    class EdgeHubRestartMessageReport : TestResultReportBase
+    {
+        public EdgeHubRestartMessageReport(
+            string trackingId,
+            string resultType,
+            bool isPassing,
+            ulong passedMessageCount,
+            string senderSource,
+            string receiverSource,
+            ulong senderMessageCount,
+            ulong receiverMessageCount,
+            Dictionary<HttpStatusCode, List<TimeSpan>> completedStatusHistogram,
+            TimeSpan minPeriod,
+            TimeSpan maxPeriod,
+            TimeSpan medianPeriod,
+            TimeSpan meanPeriod,
+            TimeSpan variancePeriod)
+            : base(trackingId, resultType)
+        {
+            this.isPassing = isPassing;
```

Comment:
isPassing should not be passing in

Reply:
Why not? I'm thinking the report itself should not know anything about the logic to determine if the test is passnig or not. I'm thinking a report as a vessel just to get the report out.